README.md

```diff
@@ -25,22 +25,36 @@ host_key_checking = False
 ```
 cd ~/src/
 git clone git@github.com:redhat-cop/casl-ansible.git
-git clone git@github.com:openshift/openshift-ansible.git
 ```
 * Copy `casl-ansible/inventory/sample.casl.example.com.d/inventory/clouds.yaml` to `~/.config/openstack/clouds.yaml`
 
+* Download/untar `openshift-ansible` for use as part of the install. See table below for versions / urls to be used for the download. Note that other versions may work as well, but these are the ones we have tested and found to be stable. Feel free to submit PRs with updated versions as they are found to be functional. 
+
+(*hint* right-click the `openshift-ansible` version number in the table below and copy the URL)
+
+```
+wget <url> -O - | tar -xz
+ln -fs openshift-ansible-*<version>* openshift-ansible
+```
+
+| openshift-ansible url     | OpenShift version | 
+|:-------------------------:|:-----------------:|
+| [3.4.50-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.4.60-1.tar.gz) | OCP 3.4 |
+
 Cool! Now you're ready to provision OpenShift clusters
 
 ### Provision a Cluster
 
-To start, we'll provision the `sample.casl.example.com` cluster defined in the `casl-ansible/inventory` directory.
+To start, we'll provision the `sample.casl.example.com` cluster defined in the `casl-ansible/inventory` directory. **Note**: It is recommended that you use a different inventory by duplicating the `sample.casl.example.com` directory and keep it elsewhere. In that way, you can update/remove/change your casl-ansble directory without loosing your inventory once used. The following is just an example.
```

Comment:
1. Can we make the **Note** a callout on it's own paragraph?

2. `loosing` should be `losing`